data/processed/aps/aps_econ_active_stem_associate_profs_data.yaml

```diff
@@ -0,0 +1,27 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: FETCH
+description_short: Associated STEM professionals
+description: Number of Science, Research, Engineering and Technology associated professionals by NUTS 2 regions.
+endpoint_url: https://www.nomisweb.co.uk/api/v01/dataset/NM_17_1.jsonstat.json?geography=1837105153...1837105192&date=latestMINUS23,latestMINUS19,latestMINUS15,latestMINUS11,latestMINUS7,latestMINUS3&cell=404882177,404883201&measures=20100,20701
+framework_group: 1
+is_experimental: False
+order: [year, nuts_id, nuts_year_spec, value.id] # do not change; specifies the order of the exported indicator fields
+region:
+  type: NutsRegion
+  level: 2
+  source: PROVIDED
+schema:
+  nuts_id:
+    type: NutsRegion.nuts_id
+  nuts_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    description: Number of Science, Research, Engineering and Technology associated professionals.
+    format: string

```

Comment:
we shouldn't use `format` for `int`s (sorry I didn't notice this in the previous pass)